Author: David Kunz <david.kunz@bfh.ch>
Description: Changing to setuptools to fix installation of /usr/bin/sphinx-autobuild.

diff -Naurp sphinx-autobuild.orig/setup.py sphinx-autobuild/setup.py
--- sphinx-autobuild.orig/setup.py
+++ sphinx-autobuild/setup.py
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
 # setup.py generated by flit for tools that don't yet use PEP 517
 
-from distutils.core import setup
+from setuptools import setup
 
 packages = \
 ['sphinx_autobuild']
